src/core/src/op/util/symbolic_info.cpp

```diff
@@ -36,30 +19,20 @@ bool ov::skip_invalidation(const ov::descriptor::Tensor& tensor) {
     return rt_info.count(type) && rt_info.at(type).as<ov::SymbolicInfo>().get_skip_invalidation();
 }
 
-std::shared_ptr<ov::TableOfEquivalence> ov::table_of_equivalence(const ov::descriptor::Tensor& tensor) {
-    const auto& rt_info = tensor.get_rt_info();
-    return get_table(rt_info);
-}
-
-std::shared_ptr<ov::TableOfEquivalence> ov::table_of_equivalence(const std::shared_ptr<ov::Model>& model) {
-    const auto& rt_info = model->get_rt_info();
-    return get_table(rt_info);
-}
-
-void ov::populate_tensor_with_missing_labels(ov::descriptor::Tensor& tensor) {
-    if (auto table = ov::table_of_equivalence(tensor)) {
-        auto label_values = tensor.get_value_label();
-        if (label_values.empty()) {
-            const auto& pshape = tensor.get_partial_shape();
-            if (pshape.is_dynamic())
-                return;
-            label_values.resize(ov::shape_size(pshape.to_shape()), ov::no_label);
-        }
-        for (auto& label : label_values)
-            if (label == ov::no_label)
-                label = table->get_next_label();
-        tensor.set_value_label(label_values);
+void ov::populate_tensor_with_missing_symbols(ov::descriptor::Tensor& tensor) {
+    if (!tensor.get_rt_info().count(ov::SymbolicInfo::get_type_info_static()))
+        return;
+    auto symbol_values = tensor.get_value_symbol();
+    if (symbol_values.empty()) {
+        const auto& pshape = tensor.get_partial_shape();
+        if (pshape.is_dynamic())
```

Comment:
I know is re-factor but should be the shape's rank checked only? 
If dimensions are intervals I think still `symbol_values` can be resized to shape rank.

Reply:
This is about values of the tensor::descriptor